routes/api.js

```diff
@@ -344,12 +267,12 @@ router.post('/request_sms', apiMiddleware(async req => {
 
     if (
         user.last_attempt_verify_phone_number &&
-    user.last_attempt_verify_phone_number.getTime() > Date.now() - (2 * 60 * 1000)
+        user.last_attempt_verify_phone_number.getTime() > Date.now() - (2 * 60 * 1000)
     ) {
         throw new ApiError({ field: 'phoneNumber', type: 'error_api_wait' });
     }
 
-    const phoneExists = await req.db.users.count({
+    const phoneExists = db.users.count({
```

Comment:
we should `await` the return of `count` right?

Reply:
yes, good catch